Update citizenship vocab.

davidlehn opened the pull request:
- Add v2 context.
  - Updated terms.
  - Add terms.
  - Change `http://schema.org/` usage to `https://schema.org/`.
- Update vocab.
- Update example.

Replaces #16, #17, #18, #19.

BigBlueHat committed:
Add height, QuantitativeValue, unitCode, & value.

These terms allow for human height expression using terminology found in
schema.org. It uses the `QuantitativeValue` class to provide `unitCode`
specification to accommodate international measurement systems.

BigBlueHat committed:
Add height to example credential.

BigBlueHat committed:
Add residence & PostalAddress Class/properties.

BigBlueHat committed:
Add filingLocation which uses PostalAddress.

BigBlueHat committed:
Add additionalName for middle names.

BigBlueHat committed:
Add formerNationality using ISO 3166-1 codes.

BigBlueHat committed:
Update example with residence & filingLocation.

BigBlueHat committed:
Add various marriage related properties.

BigBlueHat committed:
Update example with marriage properties.

davidlehn committed:
Move v1 changes to v2.

- Revert v1 changes.
- Add v2 context.
- Update example context URL.

davidlehn committed:
Simplify type usage.

davidlehn committed:
Remove .

davidlehn committed:
Use https://schema.org/.

- Update vocab docs to use https.
- Note that the v1 context still uses http.

dlongley commented on contexts/citizenship-v2.jsonld:
```diff
@@ -0,0 +1,65 @@
+    "PostalAddress": {
+      "@id": "https://schema.org/PostalAddress",
+      "@context": {
+        "addressCountry": "https://schema.org/addressCountry",
```
```suggestion
        "@protected": true,

        "addressCountry": "https://schema.org/addressCountry",
```

dlongley commented on index.html:
```diff
@@ -204,7 +205,38 @@ <h3>Example</h3>
     "birthCountry": "Bahamas",
-    "birthDate": "1958-07-17"
+    "birthDate": "1958-07-17",
+    "height": [
+      {
+        "type": "QuantitativeValue",
+        "unitCode": "ft",
+        "value": 5
+      }, {
+        "type": "QuantitativeValue",
+        "unitCode": "in",
+        "value": 11
+      }
+    ],
```
```suggestion
    "height": {
      "type": "QuantitativeValue",
      "unitCode": "in",
      "value": 71
    }],
```

davidlehn committed:
Fix height example.

davidlehn committed:
Add `PostalAddress` `@protected` flag.

Co-authored-by: Dave Longley <dlongley@digitalbazaar.com>

davidlehn committed:
Add type scope for `QuantitativeValue`.

davidlehn committed:
Use UN/CEFACT Common Code for `unitCode`.

davidlehn committed:
Update PRC modeling.

- Add marriage related properties to context.
- Add `permanentResidentCard` property.
- Add `PermanentResidentCardCredential` type.
- Use `PermanentResidentCardCredential` for a top level VC type.
- Main credential subject is Person,PermanentResident.
- Subject has a `permanentResidentCard`.
- Move PRC properties to PRC object.
- Update example and docs.

davidlehn committed:
Change `residence` to an `@id`.

davidlehn committed:
Move `lpr` properties to PRC.

davidlehn committed:
Move `name` and `description` to credential.

- `name` and `description` are added to main credential type as
  protected terms. These terms are defined by default for the
  VerifiableCredential type in VC Data Model v2 and can be removed when
  that spec version is used.

davidlehn committed:
Move image from PRC to Person.

davidlehn committed:
Update changes section.

davidlehn committed:
Update `name` and `description`.

Co-authored-by: Manu Sporny <msporny@digitalbazaar.com>

msporny commented on index.html:
```diff
@@ -204,7 +205,32 @@ <h3>Example</h3>
     "lprNumber": "999-999-999",
```
This seems to be incorrect, as does `filingLocation` ... where is the `permanentResidentCard` entry? Shouldn't some of this stuff go in the card data?
